pkgs/by-name/um/umockdev/package.nix

```diff
@@ -103,12 +112,16 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   passthru = {
-    # libgudev is needed for an optional test but it itself relies on umockdev for testing.
-    withGudev = false;
+    features = {
+      inherit withSystemd;
```

Comment:
I don't think there's any reason for `umockdev` to export this condition. It implies that any consumer of `libudev` should indicate in a `passthru` which implementation it's using, which I think is not feasible.